views/layout.erb

```diff
@@ -1,9 +1,39 @@
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
 <html xmlns="http://www.w3.org/1999/xhtml">
 <head>
-  ...
-  <link rel="stylesheet" href="stylesheets/main.css">
-  <link rel="icon" type="image/ico" href="images/smile.jpg" />
+  <link rel="icon" type="image/ico" href="images/smile.jpg"/>
```

Comment:
Image requests don't need to added in the head but rather the body except if you have your reasons. No images for the time being.